ichnaea/data/tasks.py

```diff
@@ -21,7 +21,7 @@
 )
 from ichnaea.data.validation import (
     normalized_measure_dict,
-    normalized_wifi_measure_dict,
+    normalized_wifi_dict,
     normalized_cell_measure_dict,
```

Comment:
This change looks odd, but seems to be a result of there not being a ValidWifiMeasureSchema. For cell stuff there is however a distinct ValidCellMeasureSchema. I'd rather have a distinct ValidWifiMeasureSchema class, even if it's just a placeholder, so the cell and wifi schemas and functions are consistent with each other.